odh/base/monitoring/overrides/prometheus-operator/overlays/alerts/prometheus-rules.yaml

```diff
@@ -11,14 +11,32 @@ spec:
     - name: Operate First Availability
       rules:
         - alert: Prometheus Service Down
-          expr: up{service="prometheus-operated"} == 0
+          expr: count(up{service="prometheus-operated", pod=~"prometheus-odh-monitoring.*"} == 0) > 1
```

Comment:
Should this say less than 1?

Reply:
This basically checks how many pods are down i.e. `up{service="prometheus-operated", pod=....} == 0`. Since we have 2 pods and if **both** are down we would want to be alerted on it, so hence if `count(up{service="prometheus=operated", pod=....}) == 0) > 1` would indicate that both pods are down. Does that make sense?

Reply:
Wasn't our concern also around the metrics disappearing altogether? @4n4nd^^

Reply:
ah yeah, then in that case wouldn't it be the DeadMansSwitch alert, because that essentially is to notify if prometheus itself is down and `count` will return "No Data" since the `pod` labels would be missing

Reply:
it was, I think this should work and if it does not this is still a good starting point